controllers/v1/observationSubmissionsController.js

```diff
@@ -1124,6 +1124,40 @@ module.exports = class ObservationSubmissions extends Abstract {
               throw new Error(messageConstants.apiResponses.MULTIPLE_SUBMISSIONS_NOT_ALLOWED);
             }
 
+            if(req.body.evidence.notApplicable && req.body.evidence.answers == undefined){
```

Comment:
@borkarsaish65 Can you add comment which will explain this new if block for future reference

Reply:
done